gradle/publish-dists.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'maven-publish'
```

Comment:
publishing.publishing is in `atlasdb-timelock-server/build.gradle`. I am sure it works as I tested with sandbox.